paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -2266,4 +2318,52 @@ public InlineElement getDescription() {
             return text(description);
         }
     }
+
+    /** The kms client for encryption. */
+    public enum EncryptionKmsClient implements DescribedEnum {
+        MEMORY("memory", "Use memory kms for encryption, this is only for test."),
```

Comment:
If it is only for test, why it need to be a public option?

Reply:
It's not just for test， it is a variable of the `EncryptionManager` , if the user has enabled encryption, the kms client must be specified